README.md

```diff
@@ -6,50 +6,54 @@
+* [5322](https://tools.ietf.org/html/rfc5322),
+* [6530](https://tools.ietf.org/html/rfc6530),
+* [6531](https://tools.ietf.org/html/rfc6531),
 * [6532](https://tools.ietf.org/html/rfc6532),
-* [1035](https://tools.ietf.org/html/rfc1035) 
+* [1035](https://tools.ietf.org/html/rfc1035)
 
 ## Supported versions
 
 **Current major version with full support is v3**
 
-| Version |  Released  |  EOL  | Only critical bug fixes | Full  |
-| :-----: | :--------: | :---: | :---------------------: | :---: |
-| v4.x    | 2023/01/07 |   -   |            X            |   X   |
-| v3.x    | 2020/12/29 |   -   |            X            |       |
-| v2.1.x  | 2016/05/16 |  YES  |                         |       |
-| v1.2    | 2013/19/05 |  YES  |                         |       |
+| Version |  Released  | EOL | Only critical bug fixes | Full |
+|:-------:|:----------:|:---:|:-----------------------:|:----:|
+|  v4.x   | 2023/01/07 |  -  |            X            |  X   |
+|  v3.x   | 2020/12/29 |  -  |            X            |      |
+| v2.1.x  | 2016/05/16 | YES |                         |      |
+|  v1.2   | 2013/19/05 | YES |                         |      |
 
+## Requirements
 
-## Requirements ##
+* PHP 8.1
+* [Composer](https://getcomposer.org) is required for installation
+* [Spoofchecking](/src/Validation/Extra/SpoofCheckValidation.php)
+  and [DNSCheckValidation](/src/Validation/DNSCheckValidation.php) validation requires that your PHP system has
+  the [PHP Internationalization Libraries](https://php.net/manual/en/book.intl.php) (also known as PHP Intl)
 
- * PHP 8.1
- * [Composer](https://getcomposer.org) is required for installation
- * [Spoofchecking](/src/Validation/Extra/SpoofCheckValidation.php) and [DNSCheckValidation](/src/Validation/DNSCheckValidation.php) validation requires that your PHP system has the [PHP Internationalization Libraries](https://php.net/manual/en/book.intl.php) (also known as PHP Intl)
+**Note
+**: `PHP version upgrades will happen to accomodate to the pace of major frameworks. Minor versions bumps will go via minor versions of this library (i.e: PHP7.3 -> v3.x+1). Major versions will go with major versions of the library`
 
-**Note**: `PHP version upgrades will happen to accomodate to the pace of major frameworks. Minor versions bumps will go via minor versions of this library (i.e: PHP7.3 -> v3.x+1). Major versions will go with major versions of the library`
-
-## Installation ##
+## Installation
 
 Run the command below to install via Composer
 
 ```shell
 composer require egulias/email-validator
 ```
 
-## Getting Started ##
+## Getting Started
 
-`EmailValidator` requires you to decide which (or combination of them) validation/s strategy/ies you'd like to follow for each [validation](#available-validations).
+`EmailValidator` requires you to decide which (or combination of them) validation/s strategy/ies you'd like to follow
+for each [validation](#available-validations).
```

Comment:
Please keep it on the previous line.

Reply:
Refer to https://github.com/egulias/EmailValidator/pull/364#discussion_r1212802161.